src/apps/announcements/models/announcement.py

```diff
@@ -0,0 +1,58 @@
+
+from django.db.models import (
+    Model,
+    BooleanField,
+    CharField,
+    DateTimeField,
+    IntegerField,
+    TextField,
+    BigAutoField,
+)
+from django.utils.translation import gettext_lazy as _
+
+class Announcement(Model):
+    """
+    An announcement is a block of html text for the front page display.
+    """
+
+    class Meta:
+        verbose_name = _("Announcement")
+        verbose_name_plural = _("Announcements")
+        ordering = ["-display_order"]
+
+    id = BigAutoField(primary_key=True)
+    created_at = DateTimeField(_("creation date"), auto_now_add=True, db_index=True)
+    updated_at = DateTimeField(_("updated date"), auto_now=True, db_index=True)
+    title = CharField(
+        _("title"),
+        null=False,
+        blank=False,
+        max_length=240,
+        help_text=_("Title of the announcement section."),
+        db_index=True,
```

Comment:
Do you really need an index here ?

Reply:
Unique probably implies index practically, but I removed it to just go with the default, thanks.